relayer/chains/cosmos/cosmos_chain_processor.go

```diff
@@ -41,12 +44,13 @@ func NewCosmosChainProcessor(log *zap.Logger, provider *cosmos.CosmosProvider, r
 		return nil, fmt.Errorf("error getting cosmos client: %w", err)
 	}
 	return &CosmosChainProcessor{
-		log:               log.With(zap.String("chain_id", provider.ChainId())),
-		chainProvider:     provider,
-		cc:                cc,
-		pathProcessors:    pathProcessors,
-		latestClientState: make(latestClientState),
-		channelStateCache: make(processor.ChannelStateCache),
+		log:                  log.With(zap.String("chain_id", provider.ChainId())),
```

Comment:
One thing I've been wondering is with the introduction of the chain name, as the identifier we are using for many of the CLI commands and the mapping of the chains in the config, do we want to start using the chain name for logging vs. the chain ID?

I don't think that's within the scope of this PR but something I thought was worth discussing.

Reply:
Would definitely be helpful, we could include both here.